### ISSUE TITLE
A machine learning-based choledocholithiasis prediction tool to improve ERCP decision making: a proof-of-concept study

### ISSUE THREAD

**Issue by sdiaeyu6n** [sdiaeyu6n/paper-review#2]
## [Title]
A machine learning-based choledocholithiasis prediction tool to improve ERCP decision making: a proof-of-concept study
### Quick Look
**Authors**: [Steven N Steinway](https://pubmed.ncbi.nlm.nih.gov/?term=Steinway+SN&cauthor_id=37699524) [1](https://pubmed.ncbi.nlm.nih.gov/37699524/#full-view-affiliation-1), [Bohao Tang](https://pubmed.ncbi.nlm.nih.gov/?term=Tang+B&cauthor_id=37699524) [2](https://pubmed.ncbi.nlm.nih.gov/37699524/#full-view-affiliation-2), [Jeremy Telezing](https://pubmed.ncbi.nlm.nih.gov/?term=Telezing+J&cauthor_id=37699524) [2](https://pubmed.ncbi.nlm.nih.gov/37699524/#full-view-affiliation-2), [Aditya Ashok](https://pubmed.ncbi.nlm.nih.gov/?term=Ashok+A&cauthor_id=37699524) [1](https://pubmed.ncbi.nlm.nih.gov/37699524/#full-view-affiliation-1), [Ayesha Kamal](https://pubmed.ncbi.nlm.nih.gov/?term=Kamal+A&cauthor_id=37699524) [1](https://pubmed.ncbi.nlm.nih.gov/37699524/#full-view-affiliation-1), [Chung Yao Yu](https://pubmed.ncbi.nlm.nih.gov/?term=Yu+CY&cauthor_id=37699524) [3](https://pubmed.ncbi.nlm.nih.gov/37699524/#full-view-affiliation-3), [Nitin Jagtap](https://pubmed.ncbi.nlm.nih.gov/?term=Jagtap+N&cauthor_id=37699524) [4](https://pubmed.ncbi.nlm.nih.gov/37699524/#full-view-affiliation-4), [James L Buxbaum](https://pubmed.ncbi.nlm.nih.gov/?term=Buxbaum+JL&cauthor_id=37699524) [5](https://pubmed.ncbi.nlm.nih.gov/37699524/#full-view-affiliation-5), [Joseph Elmunzer](https://pubmed.ncbi.nlm.nih.gov/?term=Elmunzer+J&cauthor_id=37699524) [6](https://pubmed.ncbi.nlm.nih.gov/37699524/#full-view-affiliation-6), [Sachin B Wani](https://pubmed.ncbi.nlm.nih.gov/?term=Wani+SB&cauthor_id=37699524) [7](https://pubmed.ncbi.nlm.nih.gov/37699524/#full-view-affiliation-7), [Mouen A Khashab](https://pubmed.ncbi.nlm.nih.gov/?term=Khashab+MA&cauthor_id=37699524) [1](https://pubmed.ncbi.nlm.nih.gov/37699524/#full-view-affiliation-1), [Brian S Caffo](https://pubmed.ncbi.nlm.nih.gov/?term=Caffo+BS&cauthor_id=37699524) [#](https://pubmed.ncbi.nlm.nih.gov/37699524/#full-view-equal-contrib-explanation) [2](https://pubmed.ncbi.nlm.nih.gov/37699524/#full-view-affiliation-2), [Venkata S Akshintala](https://pubmed.ncbi.nlm.nih.gov/?term=Akshintala+VS&cauthor_id=37699524) [#](https://pubmed.ncbi.nlm.nih.gov/37699524/#full-view-equal-contrib-explanation) [1](https://pubmed.ncbi.nlm.nih.gov/37699524/#full-view-affiliation-1)

**Date Published**: 2024 Mar;56(3):165-171. doi: 10.1055/a-2174-0534. Epub 2023 Sep 12.

**Paper Link**: https://pubmed.ncbi.nlm.nih.gov/37699524/

**Comments**: Published at NIH(National Library of Medicine)

**TLDR**: [One or at most two line summary.]

**Relevance**: 5

**Tags**: medical, prediction, decision making

## Paper summary (What)
[Summary of the paper - a few sentences with bullet points. What did they do? What is their hypothesis? ]

## Domain background
choledocholithiasis: presence of gallstones in the common bile duct (CBD)
- procedure for choledocholithiasis: ERCP with bile duct stone clearance
- cons: high cost, radiation exposure, adverse events

## Issues addressed by the paper (Why)
- limited accuracy of existing guidelines for predicting choledocholithiasis, leading to overutilization of ERCP(endoscopic retrograde cholangiopancreatography)
- appropriately select patients for ERCP

## Data
**1378 patients**
- integrated patient-level data from two large cohorts of patients with suspected choledocholithiasis
(1) Medical University of South Carolina between January 1, 2009 and December 31, 2014
(2) Los Angeles County Hospital between January 2010 and November 2016
- biliary sludge == choledocholithiasis
- initial n = 1521
- exclusion: patients with acute cholangitis(n=143)
- for effectiveness comparison, the same patients were classified as high risk according to 2010, 2019 ASGE, 2019 ESGE guidelines.

### Problem Setting
Prediction, Risk model
- reformulate patient classes and establish more accurate risk models

### Methodology
- Model: Prediction models were developed using the [gradient boosting model(GBM)](https://velog.io/@sieunc/Gradient-Boosting-Algorithm)
- Validation: 10-fold cross-validation
- Evaluation: AUC(area under the receiver operating characteristics)
- Important predictors of choledocholithiasis were identified based on relative importance in the GBM.

## Detailed Information (How)
- three models (GBM + two different data / + delta)
(1) initial liver enzyme testing (AST, ALT, ALP, total bilirubin)
(2) initial liver enzyme levels & follow-up set of test results 
-> determine whether a second set of results improved the diagnostic yield of the model
(3) incorporate delta in lab values (delta = difference between 1st and 2nd set of test values) 
- **Gini Impurity Index** was used to determine the importance of predictors in modeling classification

### Assumptions
by utilizing GBM, we can predict the need for ERCP more accurately than existing ASGE and ESGE guidelines

### Prominent Formulas
[Can be empty]

### Results
- with 1 set of variables -> 8 variables were identified as predictors of choledocholithiasis based on the decrease in the Gini Impurity Index with the inclusion of each feature.
- with 2 sets of variables -> 12 variables
<img src="https://github.com/sdiaeyu6n/paper-review/assets/89725142/83560fe1-29c9-4660-a179-6aef9268552d" width="420" height="280" />

- ultrasound was the single strongest predictor in both models
- adding one more dataset only increased the performance slightly -> chose the GBM model that uses only one set of variables (incorporation of the delta lab values worsened the performance)

- GBM accuracy: 71.5% (SC 2.5%)
- AUC 0.786 (SC 0.06)
- performed better than 2019 American Society for Gastrointestinal Endoscopy (ASGE) guidelines (accuracy 62.4% [SD 2.6%]; AUC 0.63 [SD 0.03]) and European Society of Gastrointestinal Endoscopy (ESGE) guidelines (accuracy 62.8% [SD 2.6%]; AUC 0.67 [SD 0.02]).
- sensitivity & specificity are better than 2010, 2019 ASGE guidelines (ESGE guidelines' specificity was highest but sensitivity was lowest)
- GBM had better precision(ppv) & npv than ASGE (ESGE guidelines' ppv was highest but npv was lowest)
- GBM correctly categorized 22% of patients directed to unnecessary ERCP by ASGE guidelines, and appropriately recommended as the next management step 48% of ERCPs incorrectly rejected by ESGE guidelines.
- calculator was created

### Limitations
- didn't identify a statistical significance for a **dynamic assessment** of liver function laboratory test (spontaneous stone passage, lab tests)
(1) shortcomings of study population (retrosprective)
(2) unable to include EUS and MRCP in decision making which are essential pre-ERCP screening tools
(3) cohorts had high prevalence of choledocholithiasis

### The author's conclusions
- advent of noninvasive evaluation of the biliary system toward ERCP
- represent an additional opportunity for improvement in risk stratification
- Strengths
(1) diverse datasets from different medical centers -> enhance applicability
(2) improve accuracy of prediction
calculator -> (3) user-friendly interface (4) no additional diagnostic testing, only based on routinely available parameters

### My Conclusion
- since the prediction of choledocholithiasis is a critical issue, applying ml technique to the subject of substantial medical importance was a good approach
- however, this study is much more like "medical study", not a study of machine learning model so if I want to relate this study to my own research, I should focus on making sure my study set the focus on model itself (as AI grat student).

### Rating
👍

## Possible future work / improvements
- paper suggests; include EUS and MRCP in decision making which are essential pre-ERCP screening tools
- utilize more data (not even the number of patients or sources(medical centers) but image data, or video data)

## Relation to Own Work
- What can we learn from their approach: 
  - I can try utilizing the Gini Impurity Index to measure the importance of predictors
  - number of selected predictors can range around 8-12
  - guidelines can be a good references for predictor selection
- How are we different:
  - data; multimodal; image + tabular

## Extra
- Cited references to follow up on/related papers/check google scholar for papers citing this paper:
- Source code/blog/twitter thread/other links:
## References
https://github.com/KaleabTessera/Research-Paper-Reading-Template
